docs/our_work/ds218_rap_community_of_practice.md

```diff
@@ -1,29 +1,42 @@
 ---
 layout: base
 title: NHS Reproducible Analytical Pipelines (RAP) Champion Squad
-permalink: ds218_rap_community_of_practice.html
+summary: Our roving squad of RAP champions have helped a number of teams not only transform their pipelines, but also taught them how to train others and produced guidance which is used by many other organisations.
+origin: 'NHS Digital'
+tags: ['RAP','Best Practice','Python', 'R']
 ---
 
-> | "Our roving squad of RAP champions have helped a number of teams not only transform their pipelines, but also tought them how to train others and produced guidance which is used by many other organisations."   
-
-<a href = "https://nhsdigital.github.io/rap-community-of-practice/">
-<figure markdown >
-![An image displaying the front page of the NHS RAP Community of Practice website.](../images/ds218_rap_community_of_practice.png) </a>
-<figcaption>Figure 1: The front page of the RAP Community of Practice website.</figcaption>
+<a href="https://www.linkedin.com/feed/hashtag/reproducibleanalyticalpipelines/">
+<figure markdown>
+![Multiple examples of how the team has marketed Reproducible Analytical Pipelines as a way of working, on LinkedIn, Teams, through drop-in sessions, seminars and conference talks.](../images/RAP - marketing examples.png) </a>
+<figcaption>Figure 1: Multiple examples of how the team has marketed Reproducible Analytical Pipelines as a way of working. Clicking on the image will take you through to our feed of Linkedin posts: #ReproducibleAnalyticalPipelines.</figcaption>
 </figure>
 
 
-Our Squad of RAP Champions has supported the rollout of RAP across the Analytics area within NHE England. This has involved taking existing guidance on RAP found elsewhere, and interpreting it in the local context of NHSE, making guidance specific to our systems and the problems faced by our analysts. We also put together a program for how to learn RAP, transform pipelines (through the "[thin slice approach](https://nhsdigital.github.io/rap-community-of-practice/our_RAP_service/thin-slice-strategy/)") and [become a RAP champion yourself](https://nhsdigital.github.io/rap-community-of-practice/our_RAP_service/building_team_capability/). 
+Our Squad of RAP Champions has supported the rollout of RAP across the Analytics area within NHS England. We've approached through three main pillars of work:
```

Comment:
maybe an explanation of RAP as an introduction? might not be necessary, but even just a link out to this page could be good https://nhsdigital.github.io/rap-community-of-practice/introduction_to_RAP/what_is_RAP/

Reply:
Added!